matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/presence/model/UserPresence.kt

```diff
@@ -20,4 +20,16 @@ data class UserPresence(
         val statusMessage: String? = null,
         val isCurrentlyActive: Boolean? = null,
         val presence: PresenceEnum = PresenceEnum.OFFLINE
-)
+) {
+    /**
+     * Transforms the presence to a human readable string from the given multilingual value
+     * Currently we support only online/offline (unavailable will be marked as offline)
+     */
+    fun toHumanReadable(onlineLabel: String, offlineLabel: String): String {
```

Comment:
this is weird to have this fun SDK side.
It should be better to have an Extension app side.